bundle/deployplan/plan.go

```diff
@@ -58,8 +58,9 @@ func (p *Plan) GetActions() []Action {
 	actions := make([]Action, 0, len(p.Plan))
 	for key, entry := range p.Plan {
 		at := ActionTypeFromString(entry.Action)
-		parts := strings.SplitN(strings.TrimPrefix(key, "resources."), ".", 2)
-		if len(parts) < 2 {
+		parts := strings.Split(key, ".")
+		if len(parts) == 4 {
+			// For compatibility between terraform and direct output filter out permissions and grants
```

Comment:
The previous condition tested for invalid (shorter) paths. Do we need to retain that?

Reply:
no, the worse that could happen is that you see this invalid resource in "bundle plan" but that's good for acceptance tests.